src/dataflow/utilities/node.ts

```diff
@@ -0,0 +1,126 @@
+export const NodeOperationTypes = [
+  {
+    name: "add",
+    type: "math",
+    method: (n1: number, n2: number) => n1 + n2
+  },
+  {
+    name: "subtract",
+    type: "math",
+    method: (n1: number, n2: number) => n1 - n2
+  },
+  {
+    name: "multiply",
+    type: "math",
+    method: (n1: number, n2: number) => n1 * n2
+  },
+  {
+    name: "divide",
+    type: "math",
+    method: (n1: number, n2: number) => n1 / n2
+  },
+  {
+    name: "absolute value",
+    type: "transform",
+    method: (n1: number, n2: number) => Math.abs(n1)
+  },
+  {
+    name: "negation",
+    type: "transform",
+    method: (n1: number, n2: number) => 0 - n1
+  },
+  {
+    name: "not",
+    type: "transform",
+    method: (n1: number, n2: number) => n1 ? 0 : 1
+  },
+  {
+    name: "greater than",
+    type: "logic",
+    method: (n1: number, n2: number) => +(n1 > n2)
+  },
+  {
+    name: "less than",
+    type: "logic",
+    method: (n1: number, n2: number) => +(n1 < n2)
+  },
+  {
+    name: "greater than equal to",
+    type: "logic",
+    method: (n1: number, n2: number) => +(n1 >= n2)
+  },
+  {
+    name: "less than equal to",
+    type: "logic",
+    method: (n1: number, n2: number) => +(n1 <= n2)
+  },
+  {
+    name: "equal",
+    type: "logic",
+    method: (n1: number, n2: number) => +(n1 === n2)
+  },
+  {
+    name: "not equal",
+    type: "logic",
+    method: (n1: number, n2: number) => +(n1 !== n2)
+  },
+  {
+    name: "and",
+    type: "logic",
+    method: (n1: number, n2: number) => n1 && n2 ? 1 : 0
+  },
+  {
+    name: "or",
+    type: "logic",
+    method: (n1: number, n2: number) => n1 || n2 ? 1 : 0
+  },
+  {
+    name: "nand",
+    type: "logic",
+    method: (n1: number, n2: number) => +(!(n1 && n2 ? 1 : 0))
+  },
+  {
+    name: "xor",
+    type: "logic",
+    method: (n1: number, n2: number) => +((n1 ? 1 : 0) !== (n2 ? 1 : 0))
+  }
+];
+
+export const NodeSensorTypes = [
+  {
+    name: "temperature",
+    units: "°C",
+  },
+  {
+    name: "humidity",
+    units: "%",
+  },
+  {
+    name: "CO2",
```

Comment:
If this is user-visible, would it make sense to use a subscript here, e.g. "CO₂" and "O₂" (below)?

Reply:
Good call.  I was using the DF 2.0 names (which for whatever reason didn't include subscripts), but using the actual subscripts is much better.  Fixed.